datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/readmes.py

```diff
@@ -79,6 +87,10 @@ def readmes(ctx, integration):
             errors = True
             display_queue.append((echo_failure, "     readme does not contain both an Overview and Setup H2 section"))
 
+        if not has_support and support_type == 'partner':
+            errors = True
+            display_queue.append((echo_failure, "     readme does not contain a Support H2 section"))
```

Comment:
Why so much space for the string?

Reply:
Its because the root info message starts with zero indentation and states the integration name, then all following errors get indented to visually show that error belongs to that integration.

Here's an example of the output when the command finds an invalid readme:
```╰─ ddev -x validate readmes
<integration_name>/README.md... FAILED
     readme does not contain a Support H2 section
1 invalid files```